extensions/arc/src/ui/dashboards/postgres/postgresCoordinatorNodeParametersPage.ts

```diff
@@ -0,0 +1,595 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Given that the two parameters pages are nearly identical it would be nice to have a base "Parameters" page that each of them just then inherit from and customize as needed.